### PR TITLE
Refactory(Activity): Addition of activity details and Activity's refactoring

### PR DESCRIPTION
**Pull request by Marielbamar21**
## PR Checklist
Please check if your PR fulfills the following requirements:

- [ ] The commit message follows our guidelines: https://github.com/kumawallet/extension/blob/master/CONTRIBUTING.md
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ X ] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [X ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Other... Please describe:

## What is the new behavior?
The activity table was adapted to Figma and "Activity Details" was also added. Minor functional modifications were made, including the addition of a new field to the records (isSwap) to identify swaps, as well as the incorporation of small functions that manage how the data is displayed.

## Does this PR introduce a breaking change?

- [ ] Yes
- [ X ] No

### CODE REVIEW

File: src/pages/balance/components/Activity.tsx [hunk 1]
```diff
@@ -117,85 +132,126 @@ export const Activity = () => {
...
+              setSearch(e.target.value);
+            }}
+          />
+          <CiSearch className={stylesActivity.iconSearch} />
+      </div>
+      <div className={stylesActivity.countainerTx}>
         {activity.length === 0 && (
-          <div className="flex justify-center items-center mt-5">
-            <p className="text-lg font-medium">{t("empty")}</p>
+          <div className={` ${stylesActivity.flexItemsCenter}${stylesActivity.countainerEmptyActivity}`}>
```
Comment (rubenguc) [inline src/pages/balance/components/Activity.tsx:159]: This is generating a className like this:

```
className="flex items-centerjustify-center mt-5"
```

add a space between this twos classes:
```
`${stylesActivity.flexItemsCenter} ${stylesActivity.countainerEmptyActivity}`
```

File: src/pages/balance/components/Activity.tsx [hunk 2]
```diff
@@ -117,85 +132,126 @@ export const Activity = () => {
...
+                                                                  address: address,
+                                                                  network: network,
+                                                                  recipientNetwork: recipientNetwork,
+                                                                  data: data,
+                                                                  type: type,
+              }})}
+              
             >
-              <div className="flex items-center justify-between gap-3">
+              <div className={`${stylesActivity.flexItemsCenter}${stylesActivity.countainerButton}`}>
```
Comment (rubenguc) [inline src/pages/balance/components/Activity.tsx:192]: same here, separate add  a space

**Review by rubenguc — CHANGES_REQUESTED** [review]

File: src/components/common/Button.tsx [hunk 3]
```diff
@@ -14,7 +14,7 @@ interface ButtonProps {
   onClick?: () => void;
   spinnerClassname?: SVGAttributes<SVGElement>["className"];
   style?: CSSProperties;
-  variant?: "text" | "outlined" | "contained" | "contained-gray" | "contained-black";
+  variant?: "text" | "outlined" | "contained" | "contained-gray" | "contained-black"| "contained-litlle-gray";
```
[the remark this answers is not in the document]
Comment (Marielbamar21) [reply src/components/common/Button.tsx]: done

File: src/components/common/Button.tsx [hunk 4]
```diff
@@ -25,6 +25,8 @@ const getVariantStyles = (
   variant: ButtonProps["variant"],
 ) => {
   switch (variant) {
+    case "contained-litlle-gray":
```
[the remark this answers is not in the document]
Comment (Marielbamar21) [reply src/components/common/Button.tsx]: done

File: src/pages/balance/components/Activity.tsx (hunk 1 above)
Comment (Marielbamar21) [reply src/pages/balance/components/Activity.tsx]: done

File: src/pages/balance/components/Activity.tsx (hunk 2 above)
Comment (Marielbamar21) [reply src/pages/balance/components/Activity.tsx]: done

**Pull request merged**